pkg/push/push.proto

```diff
@@ -0,0 +1,40 @@
+syntax = "proto3";
+
+package push;
```

Comment:
To address breaking concerns, I wonder if we can name this with the same package as before: `package logproto;`

Reply:
I need the `push` package to be a separate module, so I can import it outside of loki. That's why it has its own name.
If push would be a part of logproto package then I would need to import the whole logproto package from the outside.
That's will be ok, but logproto can't be a separate module right now, because it has dependencies on other loki packages:
github.com/grafana/loki/pkg/querier/queryrange/queryrangebase/definitions
github.com/grafana/loki/pkg/util
github.com/grafana/loki/pkg/storage/stores/tsdb/index
github.com/grafana/loki/pkg/logqlmodel/stats

To make logproto a separate module I would need to remove these dependencies from it. As I understood that dependencies were brought to logproto on purpose

Reply:
it seems working! renamed package from `push` to `logproto`